### ISSUE TITLE
"lora key not loaded: lora_te1_text_model_encoder_layers" Message

### ISSUE THREAD

**Issue by LearningCreator** [cocktailpeanut/fluxgym#110]
I trained two loras without changing any of the default parameters using the latest version of FluxGym. When using them with a simple workflow in ComfyUI (also the newest version), they work but I always get a wall of "lora key not loaded" as shown below. Am I doing something wrong or should I ignore the message?

`got prompt
Using pytorch attention in VAE
Using pytorch attention in VAE
model weight dtype torch.float8_e4m3fn, manual cast: torch.bfloat16
model_type FLUX
lora key not loaded: lora_te1_text_model_encoder_layers_0_mlp_fc1.alpha
lora key not loaded: lora_te1_text_model_encoder_layers_0_mlp_fc1.lora_down.weight
lora key not loaded: lora_te1_text_model_encoder_layers_0_mlp_fc1.lora_up.weight
lora key not loaded: lora_te1_text_model_encoder_layers_0_mlp_fc2.alpha
lora key not loaded: lora_te1_text_model_encoder_layers_0_mlp_fc2.lora_down.weight
lora key not loaded: lora_te1_text_model_encoder_layers_0_mlp_fc2.lora_up.weight
lora key not loaded: lora_te1_text_model_encoder_layers_0_self_attn_k_proj.alpha
lora key not loaded: lora_te1_text_model_encoder_layers_0_self_attn_k_proj.lora_down.weight
lora key not loaded: lora_te1_text_model_encoder_layers_0_self_attn_k_proj.lora_up.weight
lora key not loaded: lora_te1_text_model_encoder_layers_0_self_attn_out_proj.alpha
lora key not loaded: lora_te1_text_model_encoder_layers_0_self_attn_out_proj.lora_down.weight
lora key not loaded: lora_te1_text_model_encoder_layers_0_self_attn_out_proj.lora_up.weight
lora key not loaded: lora_te1_text_model_encoder_layers_0_self_attn_q_proj.alpha
lora key not loaded: lora_te1_text_model_encoder_layers_0_self_attn_q_proj.lora_down.weight
lora key not loaded: lora_te1_text_model_encoder_layers_0_self_attn_q_proj.lora_up.weight
lora key not loaded: lora_te1_text_model_encoder_layers_0_self_attn_v_proj.alpha
lora key not loaded: lora_te1_text_model_encoder_layers_0_self_attn_v_proj.lora_down.weight
lora key not loaded: lora_te1_text_model_encoder_layers_0_self_attn_v_proj.lora_up.weight
lora key not loaded: lora_te1_text_model_encoder_layers_10_mlp_fc1.alpha
lora key not loaded: lora_te1_text_model_encoder_layers_10_mlp_fc1.lora_down.weight
lora key not loaded: lora_te1_text_model_encoder_layers_10_mlp_fc1.lora_up.weight
lora key not loaded: lora_te1_text_model_encoder_layers_10_mlp_fc2.alpha
lora key not loaded: lora_te1_text_model_encoder_layers_10_mlp_fc2.lora_down.weight
lora key not loaded: lora_te1_text_model_encoder_layers_10_mlp_fc2.lora_up.weight
lora key not loaded: lora_te1_text_model_encoder_layers_10_self_attn_k_proj.alpha
lora key not loaded: lora_te1_text_model_encoder_layers_10_self_attn_k_proj.lora_down.weight
lora key not loaded: lora_te1_text_model_encoder_layers_10_self_attn_k_proj.lora_up.weight
lora key not loaded: lora_te1_text_model_encoder_layers_10_self_attn_out_proj.alpha
lora key not loaded: lora_te1_text_model_encoder_layers_10_self_attn_out_proj.lora_down.weight
lora key not loaded: lora_te1_text_model_encoder_layers_10_self_attn_out_proj.lora_up.weight
lora key not loaded: lora_te1_text_model_encoder_layers_10_self_attn_q_proj.alpha
lora key not loaded: lora_te1_text_model_encoder_layers_10_self_attn_q_proj.lora_down.weight
lora key not loaded: lora_te1_text_model_encoder_layers_10_self_attn_q_proj.lora_up.weight
lora key not loaded: lora_te1_text_model_encoder_layers_10_self_attn_v_proj.alpha
lora key not loaded: lora_te1_text_model_encoder_layers_10_self_attn_v_proj.lora_down.weight
lora key not loaded: lora_te1_text_model_encoder_layers_10_self_attn_v_proj.lora_up.weight
lora key not loaded: lora_te1_text_model_encoder_layers_11_mlp_fc1.alpha
lora key not loaded: lora_te1_text_model_encoder_layers_11_mlp_fc1.lora_down.weight
lora key not loaded: lora_te1_text_model_encoder_layers_11_mlp_fc1.lora_up.weight
lora key not loaded: lora_te1_text_model_encoder_layers_11_mlp_fc2.alpha
lora key not loaded: lora_te1_text_model_encoder_layers_11_mlp_fc2.lora_down.weight
lora key not loaded: lora_te1_text_model_encoder_layers_11_mlp_fc2.lora_up.weight
lora key not loaded: lora_te1_text_model_encoder_layers_11_self_attn_k_proj.alpha
lora key not loaded: lora_te1_text_model_encoder_layers_11_self_attn_k_proj.lora_down.weight
lora key not loaded: lora_te1_text_model_encoder_layers_11_self_attn_k_proj.lora_up.weight
lora key not loaded: lora_te1_text_model_encoder_layers_11_self_attn_out_proj.alpha
lora key not loaded: lora_te1_text_model_encoder_layers_11_self_attn_out_proj.lora_down.weight
lora key not loaded: lora_te1_text_model_encoder_layers_11_self_attn_out_proj.lora_up.weight
lora key not loaded: lora_te1_text_model_encoder_layers_11_self_attn_q_proj.alpha
lora key not loaded: lora_te1_text_model_encoder_layers_11_self_attn_q_proj.lora_down.weight
lora key not loaded: lora_te1_text_model_encoder_layers_11_self_attn_q_proj.lora_up.weight
lora key not loaded: lora_te1_text_model_encoder_layers_11_self_attn_v_proj.alpha
lora key not loaded: lora_te1_text_model_encoder_layers_11_self_attn_v_proj.lora_down.weight
lora key not loaded: lora_te1_text_model_encoder_layers_11_self_attn_v_proj.lora_up.weight
lora key not loaded: lora_te1_text_model_encoder_layers_1_mlp_fc1.alpha
lora key not loaded: lora_te1_text_model_encoder_layers_1_mlp_fc1.lora_down.weight
lora key not loaded: lora_te1_text_model_encoder_layers_1_mlp_fc1.lora_up.weight
lora key not loaded: lora_te1_text_model_encoder_layers_1_mlp_fc2.alpha
lora key not loaded: lora_te1_text_model_encoder_layers_1_mlp_fc2.lora_down.weight
lora key not loaded: lora_te1_text_model_encoder_layers_1_mlp_fc2.lora_up.weight
lora key not loaded: lora_te1_text_model_encoder_layers_1_self_attn_k_proj.alpha
lora key not loaded: lora_te1_text_model_encoder_layers_1_self_attn_k_proj.lora_down.weight
lora key not loaded: lora_te1_text_model_encoder_layers_1_self_attn_k_proj.lora_up.weight
lora key not loaded: lora_te1_text_model_encoder_layers_1_self_attn_out_proj.alpha
lora key not loaded: lora_te1_text_model_encoder_layers_1_self_attn_out_proj.lora_down.weight
lora key not loaded: lora_te1_text_model_encoder_layers_1_self_attn_out_proj.lora_up.weight
lora key not loaded: lora_te1_text_model_encoder_layers_1_self_attn_q_proj.alpha
lora key not loaded: lora_te1_text_model_encoder_layers_1_self_attn_q_proj.lora_down.weight
lora key not loaded: lora_te1_text_model_encoder_layers_1_self_attn_q_proj.lora_up.weight
lora key not loaded: lora_te1_text_model_encoder_layers_1_self_attn_v_proj.alpha
lora key not loaded: lora_te1_text_model_encoder_layers_1_self_attn_v_proj.lora_down.weight
lora key not loaded: lora_te1_text_model_encoder_layers_1_self_attn_v_proj.lora_up.weight
lora key not loaded: lora_te1_text_model_encoder_layers_2_mlp_fc1.alpha
lora key not loaded: lora_te1_text_model_encoder_layers_2_mlp_fc1.lora_down.weight
lora key not loaded: lora_te1_text_model_encoder_layers_2_mlp_fc1.lora_up.weight
lora key not loaded: lora_te1_text_model_encoder_layers_2_mlp_fc2.alpha
lora key not loaded: lora_te1_text_model_encoder_layers_2_mlp_fc2.lora_down.weight
lora key not loaded: lora_te1_text_model_encoder_layers_2_mlp_fc2.lora_up.weight
lora key not loaded: lora_te1_text_model_encoder_layers_2_self_attn_k_proj.alpha
lora key not loaded: lora_te1_text_model_encoder_layers_2_self_attn_k_proj.lora_down.weight
lora key not loaded: lora_te1_text_model_encoder_layers_2_self_attn_k_proj.lora_up.weight
lora key not loaded: lora_te1_text_model_encoder_layers_2_self_attn_out_proj.alpha
lora key not loaded: lora_te1_text_model_encoder_layers_2_self_attn_out_proj.lora_down.weight
lora key not loaded: lora_te1_text_model_encoder_layers_2_self_attn_out_proj.lora_up.weight
lora key not loaded: lora_te1_text_model_encoder_layers_2_self_attn_q_proj.alpha
lora key not loaded: lora_te1_text_model_encoder_layers_2_self_attn_q_proj.lora_down.weight
lora key not loaded: lora_te1_text_model_encoder_layers_2_self_attn_q_proj.lora_up.weight
lora key not loaded: lora_te1_text_model_encoder_layers_2_self_attn_v_proj.alpha
lora key not loaded: lora_te1_text_model_encoder_layers_2_self_attn_v_proj.lora_down.weight
lora key not loaded: lora_te1_text_model_encoder_layers_2_self_attn_v_proj.lora_up.weight
lora key not loaded: lora_te1_text_model_encoder_layers_3_mlp_fc1.alpha
lora key not loaded: lora_te1_text_model_encoder_layers_3_mlp_fc1.lora_down.weight
lora key not loaded: lora_te1_text_model_encoder_layers_3_mlp_fc1.lora_up.weight
lora key not loaded: lora_te1_text_model_encoder_layers_3_mlp_fc2.alpha
lora key not loaded: lora_te1_text_model_encoder_layers_3_mlp_fc2.lora_down.weight
lora key not loaded: lora_te1_text_model_encoder_layers_3_mlp_fc2.lora_up.weight
lora key not loaded: lora_te1_text_model_encoder_layers_3_self_attn_k_proj.alpha
lora key not loaded: lora_te1_text_model_encoder_layers_3_self_attn_k_proj.lora_down.weight
lora key not loaded: lora_te1_text_model_encoder_layers_3_self_attn_k_proj.lora_up.weight
lora key not loaded: lora_te1_text_model_encoder_layers_3_self_attn_out_proj.alpha
lora key not loaded: lora_te1_text_model_encoder_layers_3_self_attn_out_proj.lora_down.weight
lora key not loaded: lora_te1_text_model_encoder_layers_3_self_attn_out_proj.lora_up.weight
lora key not loaded: lora_te1_text_model_encoder_layers_3_self_attn_q_proj.alpha
lora key not loaded: lora_te1_text_model_encoder_layers_3_self_attn_q_proj.lora_down.weight
lora key not loaded: lora_te1_text_model_encoder_layers_3_self_attn_q_proj.lora_up.weight
lora key not loaded: lora_te1_text_model_encoder_layers_3_self_attn_v_proj.alpha
lora key not loaded: lora_te1_text_model_encoder_layers_3_self_attn_v_proj.lora_down.weight
lora key not loaded: lora_te1_text_model_encoder_layers_3_self_attn_v_proj.lora_up.weight
lora key not loaded: lora_te1_text_model_encoder_layers_4_mlp_fc1.alpha
lora key not loaded: lora_te1_text_model_encoder_layers_4_mlp_fc1.lora_down.weight
lora key not loaded: lora_te1_text_model_encoder_layers_4_mlp_fc1.lora_up.weight
lora key not loaded: lora_te1_text_model_encoder_layers_4_mlp_fc2.alpha
lora key not loaded: lora_te1_text_model_encoder_layers_4_mlp_fc2.lora_down.weight
lora key not loaded: lora_te1_text_model_encoder_layers_4_mlp_fc2.lora_up.weight
lora key not loaded: lora_te1_text_model_encoder_layers_4_self_attn_k_proj.alpha
lora key not loaded: lora_te1_text_model_encoder_layers_4_self_attn_k_proj.lora_down.weight
lora key not loaded: lora_te1_text_model_encoder_layers_4_self_attn_k_proj.lora_up.weight
lora key not loaded: lora_te1_text_model_encoder_layers_4_self_attn_out_proj.alpha
lora key not loaded: lora_te1_text_model_encoder_layers_4_self_attn_out_proj.lora_down.weight
lora key not loaded: lora_te1_text_model_encoder_layers_4_self_attn_out_proj.lora_up.weight
lora key not loaded: lora_te1_text_model_encoder_layers_4_self_attn_q_proj.alpha
lora key not loaded: lora_te1_text_model_encoder_layers_4_self_attn_q_proj.lora_down.weight
lora key not loaded: lora_te1_text_model_encoder_layers_4_self_attn_q_proj.lora_up.weight
lora key not loaded: lora_te1_text_model_encoder_layers_4_self_attn_v_proj.alpha
lora key not loaded: lora_te1_text_model_encoder_layers_4_self_attn_v_proj.lora_down.weight
lora key not loaded: lora_te1_text_model_encoder_layers_4_self_attn_v_proj.lora_up.weight
lora key not loaded: lora_te1_text_model_encoder_layers_5_mlp_fc1.alpha
lora key not loaded: lora_te1_text_model_encoder_layers_5_mlp_fc1.lora_down.weight
lora key not loaded: lora_te1_text_model_encoder_layers_5_mlp_fc1.lora_up.weight
lora key not loaded: lora_te1_text_model_encoder_layers_5_mlp_fc2.alpha
lora key not loaded: lora_te1_text_model_encoder_layers_5_mlp_fc2.lora_down.weight
lora key not loaded: lora_te1_text_model_encoder_layers_5_mlp_fc2.lora_up.weight
lora key not loaded: lora_te1_text_model_encoder_layers_5_self_attn_k_proj.alpha
lora key not loaded: lora_te1_text_model_encoder_layers_5_self_attn_k_proj.lora_down.weight
lora key not loaded: lora_te1_text_model_encoder_layers_5_self_attn_k_proj.lora_up.weight
lora key not loaded: lora_te1_text_model_encoder_layers_5_self_attn_out_proj.alpha
lora key not loaded: lora_te1_text_model_encoder_layers_5_self_attn_out_proj.lora_down.weight
lora key not loaded: lora_te1_text_model_encoder_layers_5_self_attn_out_proj.lora_up.weight
lora key not loaded: lora_te1_text_model_encoder_layers_5_self_attn_q_proj.alpha
lora key not loaded: lora_te1_text_model_encoder_layers_5_self_attn_q_proj.lora_down.weight
lora key not loaded: lora_te1_text_model_encoder_layers_5_self_attn_q_proj.lora_up.weight
lora key not loaded: lora_te1_text_model_encoder_layers_5_self_attn_v_proj.alpha
lora key not loaded: lora_te1_text_model_encoder_layers_5_self_attn_v_proj.lora_down.weight
lora key not loaded: lora_te1_text_model_encoder_layers_5_self_attn_v_proj.lora_up.weight
lora key not loaded: lora_te1_text_model_encoder_layers_6_mlp_fc1.alpha
lora key not loaded: lora_te1_text_model_encoder_layers_6_mlp_fc1.lora_down.weight
lora key not loaded: lora_te1_text_model_encoder_layers_6_mlp_fc1.lora_up.weight
lora key not loaded: lora_te1_text_model_encoder_layers_6_mlp_fc2.alpha
lora key not loaded: lora_te1_text_model_encoder_layers_6_mlp_fc2.lora_down.weight
lora key not loaded: lora_te1_text_model_encoder_layers_6_mlp_fc2.lora_up.weight
lora key not loaded: lora_te1_text_model_encoder_layers_6_self_attn_k_proj.alpha
lora key not loaded: lora_te1_text_model_encoder_layers_6_self_attn_k_proj.lora_down.weight
lora key not loaded: lora_te1_text_model_encoder_layers_6_self_attn_k_proj.lora_up.weight
lora key not loaded: lora_te1_text_model_encoder_layers_6_self_attn_out_proj.alpha
lora key not loaded: lora_te1_text_model_encoder_layers_6_self_attn_out_proj.lora_down.weight
lora key not loaded: lora_te1_text_model_encoder_layers_6_self_attn_out_proj.lora_up.weight
lora key not loaded: lora_te1_text_model_encoder_layers_6_self_attn_q_proj.alpha
lora key not loaded: lora_te1_text_model_encoder_layers_6_self_attn_q_proj.lora_down.weight
lora key not loaded: lora_te1_text_model_encoder_layers_6_self_attn_q_proj.lora_up.weight
lora key not loaded: lora_te1_text_model_encoder_layers_6_self_attn_v_proj.alpha
lora key not loaded: lora_te1_text_model_encoder_layers_6_self_attn_v_proj.lora_down.weight
lora key not loaded: lora_te1_text_model_encoder_layers_6_self_attn_v_proj.lora_up.weight
lora key not loaded: lora_te1_text_model_encoder_layers_7_mlp_fc1.alpha
lora key not loaded: lora_te1_text_model_encoder_layers_7_mlp_fc1.lora_down.weight
lora key not loaded: lora_te1_text_model_encoder_layers_7_mlp_fc1.lora_up.weight
lora key not loaded: lora_te1_text_model_encoder_layers_7_mlp_fc2.alpha
lora key not loaded: lora_te1_text_model_encoder_layers_7_mlp_fc2.lora_down.weight
lora key not loaded: lora_te1_text_model_encoder_layers_7_mlp_fc2.lora_up.weight
lora key not loaded: lora_te1_text_model_encoder_layers_7_self_attn_k_proj.alpha
lora key not loaded: lora_te1_text_model_encoder_layers_7_self_attn_k_proj.lora_down.weight
lora key not loaded: lora_te1_text_model_encoder_layers_7_self_attn_k_proj.lora_up.weight
lora key not loaded: lora_te1_text_model_encoder_layers_7_self_attn_out_proj.alpha
lora key not loaded: lora_te1_text_model_encoder_layers_7_self_attn_out_proj.lora_down.weight
lora key not loaded: lora_te1_text_model_encoder_layers_7_self_attn_out_proj.lora_up.weight
lora key not loaded: lora_te1_text_model_encoder_layers_7_self_attn_q_proj.alpha
lora key not loaded: lora_te1_text_model_encoder_layers_7_self_attn_q_proj.lora_down.weight
lora key not loaded: lora_te1_text_model_encoder_layers_7_self_attn_q_proj.lora_up.weight
lora key not loaded: lora_te1_text_model_encoder_layers_7_self_attn_v_proj.alpha
lora key not loaded: lora_te1_text_model_encoder_layers_7_self_attn_v_proj.lora_down.weight
lora key not loaded: lora_te1_text_model_encoder_layers_7_self_attn_v_proj.lora_up.weight
lora key not loaded: lora_te1_text_model_encoder_layers_8_mlp_fc1.alpha
lora key not loaded: lora_te1_text_model_encoder_layers_8_mlp_fc1.lora_down.weight
lora key not loaded: lora_te1_text_model_encoder_layers_8_mlp_fc1.lora_up.weight
lora key not loaded: lora_te1_text_model_encoder_layers_8_mlp_fc2.alpha
lora key not loaded: lora_te1_text_model_encoder_layers_8_mlp_fc2.lora_down.weight
lora key not loaded: lora_te1_text_model_encoder_layers_8_mlp_fc2.lora_up.weight
lora key not loaded: lora_te1_text_model_encoder_layers_8_self_attn_k_proj.alpha
lora key not loaded: lora_te1_text_model_encoder_layers_8_self_attn_k_proj.lora_down.weight
lora key not loaded: lora_te1_text_model_encoder_layers_8_self_attn_k_proj.lora_up.weight
lora key not loaded: lora_te1_text_model_encoder_layers_8_self_attn_out_proj.alpha
lora key not loaded: lora_te1_text_model_encoder_layers_8_self_attn_out_proj.lora_down.weight
lora key not loaded: lora_te1_text_model_encoder_layers_8_self_attn_out_proj.lora_up.weight
lora key not loaded: lora_te1_text_model_encoder_layers_8_self_attn_q_proj.alpha
lora key not loaded: lora_te1_text_model_encoder_layers_8_self_attn_q_proj.lora_down.weight
lora key not loaded: lora_te1_text_model_encoder_layers_8_self_attn_q_proj.lora_up.weight
lora key not loaded: lora_te1_text_model_encoder_layers_8_self_attn_v_proj.alpha
lora key not loaded: lora_te1_text_model_encoder_layers_8_self_attn_v_proj.lora_down.weight
lora key not loaded: lora_te1_text_model_encoder_layers_8_self_attn_v_proj.lora_up.weight
lora key not loaded: lora_te1_text_model_encoder_layers_9_mlp_fc1.alpha
lora key not loaded: lora_te1_text_model_encoder_layers_9_mlp_fc1.lora_down.weight
lora key not loaded: lora_te1_text_model_encoder_layers_9_mlp_fc1.lora_up.weight
lora key not loaded: lora_te1_text_model_encoder_layers_9_mlp_fc2.alpha
lora key not loaded: lora_te1_text_model_encoder_layers_9_mlp_fc2.lora_down.weight
lora key not loaded: lora_te1_text_model_encoder_layers_9_mlp_fc2.lora_up.weight
lora key not loaded: lora_te1_text_model_encoder_layers_9_self_attn_k_proj.alpha
lora key not loaded: lora_te1_text_model_encoder_layers_9_self_attn_k_proj.lora_down.weight
lora key not loaded: lora_te1_text_model_encoder_layers_9_self_attn_k_proj.lora_up.weight
lora key not loaded: lora_te1_text_model_encoder_layers_9_self_attn_out_proj.alpha
lora key not loaded: lora_te1_text_model_encoder_layers_9_self_attn_out_proj.lora_down.weight
lora key not loaded: lora_te1_text_model_encoder_layers_9_self_attn_out_proj.lora_up.weight
lora key not loaded: lora_te1_text_model_encoder_layers_9_self_attn_q_proj.alpha
lora key not loaded: lora_te1_text_model_encoder_layers_9_self_attn_q_proj.lora_down.weight
lora key not loaded: lora_te1_text_model_encoder_layers_9_self_attn_q_proj.lora_up.weight
lora key not loaded: lora_te1_text_model_encoder_layers_9_self_attn_v_proj.alpha
lora key not loaded: lora_te1_text_model_encoder_layers_9_self_attn_v_proj.lora_down.weight
lora key not loaded: lora_te1_text_model_encoder_layers_9_self_attn_v_proj.lora_up.weight
Requested to load FluxClipModel_
Loading 1 new model
loaded completely 0.0 9319.23095703125 True
clip missing: ['text_projection.weight']
Requested to load Flux
Loading 1 new model
loaded completely 0.0 11350.048889160156 True
42it [00:55,  1.31s/it]
Requested to load AutoencodingEngine
Loading 1 new model
loaded completely 0.0 159.87335777282715 True
Prompt executed in 97.93 seconds`

**Comment by 6Morpheus6** [comment]
@LearningCreator 
This wall of text means that the Lora can't be used with the model you are trying to load.
For example if you use a Flux model and an SDXL Lora, you would get this text as well.
I can't tell if it's your LORA or your workflow.
Please post your workflow and the full name of the Flux model you were using.

**Comment by LearningCreator** [comment]
Thanks for the quick response!
As I mentioned above, the lora _is_ working and it's working with both flux1-dev and flux1-schnell.
I just ran multiple gens with both models (flux1-dev and flux1-schnell) and found that the `lora key not loaded` message only appears the first time I run the queue after loading the the models and/or either of the loras I trained.
I assume that it's a "don't pay attention to it" message but I've added a screenshot of my workflow, just in case you still wanted to see it.
![wf1](https://github.com/user-attachments/assets/c0712b90-daa4-4407-a04a-90421e0cdaa0)

Thanks for the help!
